catalog/src/main/kotlin/org/openapitools/client/apis/AlbumJSONAPI.kt

```diff
@@ -0,0 +1,570 @@
+/**
+ *
+ * Please note:
+ * This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * Do not edit this file manually.
+ *
+ */
+
+@file:Suppress(
+    "ArrayInDataClass",
+    "EnumEntryName",
+    "RemoveRedundantQualifierName",
+    "UnusedImport"
+)
+
+package org.openapitools.client.apis
```

Comment:
Should we use our packagename instead?

Reply:
Yes. I couldn't convince the tool to do that without issues though. So for this first version, I want to use theirs. 

I intend to update this later, however.